src/TabsTab.jsx

```diff
@@ -34,7 +34,10 @@ class TabsTab extends React.Component {
 }
 TabsTab.propTypes = {
 	url: React.PropTypes.string.isRequired,
-	label: React.PropTypes.string.isRequired,
+	label: React.PropTypes.oneOfType([
+		React.PropTypes.string,
+		React.PropTypes.element,
```

Comment:
This looks good to me, in reference to the [ticket](https://meetup.atlassian.net/browse/WC-25) we would just add more types as needed later?